src/vs/workbench/contrib/debug/browser/debugService.ts

```diff
@@ -432,7 +433,23 @@ export class DebugService implements IDebugService {
 							nls.localize('debugTypeMissing', "Missing property 'type' for the chosen launch configuration.");
 					}
 
-					await this.showError(message);
+					let actionList: IAction[] = [];
+					let action = {
+						id: 'Install Debug Extension',
+						label: 'Install Debug Extension(s)',
+						tooltip: 'Install Debug Extension',
+						class: 'string',
+						checked: true,
+						run: async () => {
+							const viewlet = (await this.viewletService.openViewlet(EXTENSIONS_VIEWLET_ID, true))?.getViewPaneContainer() as IExtensionsViewPaneContainer;
+							viewlet.search('tag:debuggers @sort:installs');
+							return this.viewletService.openViewlet(EXTENSIONS_VIEWLET_ID, true);
+						},
+						dispose: () => this.viewletService.openViewlet(EXTENSIONS_VIEWLET_ID, true),
```

Comment:
No need for disposing, disposing is done when the action is done for, and in the dispose we generaly remove listeners, since your action does not have listeners it should be a no-op

Reply:
Makes sense- in the new way I'm defining the action now, I've removed any disposing.